kodilla-testing2/src/test/java/com/kodilla/testing2/crudapp/CruddAppTestSuite.java

```diff
@@ -105,7 +106,33 @@ private boolean checkTaskExistsInTrello(String taskName) throws InterruptedExcep
     @Test
     public void shouldCreateTrelloCard() throws InterruptedException {
         String taskName = createCrudAppTestTask();
-        sendTestTaskToTrello(taskName);
-        assertTrue(checkTaskExistsInTrello(taskName));
+        //sendTestTaskToTrello(taskName);
+        //assertTrue(checkTaskExistsInTrello(taskName));
+        assertTrue(shouldDeleteTrelloCard(taskName));
     }
-}
+
+    @Test
+    public boolean shouldDeleteTrelloCard(String taskName) throws InterruptedException {
+
+        final String XPATH_TASK_BY_NAME = "//p[contains(@data-task-name-paragraph,\"" + taskName + "\")]";
+
+        driver.navigate().refresh();
+
+        Thread.sleep(4000);
+
+        while (!driver.findElement(By.xpath("//select[1]")).isDisplayed());
+
+        driver.findElements(By.xpath("//form[@class=\"datatable__row\"]")).stream()
+                .filter(anyForm ->
```

Comment:
Tutaj stream wypada mało naturalnie.
Jeśli już chcesz go koniecznie użyć, to szukasz tutaj tak na prawdę jednego elementu - skorzystaj z https://www.baeldung.com/java-stream-findfirst-vs-findany i potem na `Optional`-u użyj `ifPresent`.